src/App.js

```diff
@@ -34,12 +19,12 @@ const App = () => {
   return (
     <div className="boxes">
       <div className="box">
-        <Header car={state.car} />
-        <AddedFeatures car={state.car} />
+        <Header />
+        <AddedFeatures />
       </div>
       <div className="box">
-        <AdditionalFeatures store={state.store} />
-        <Total car={state.car} additionalPrice={state.additionalPrice} />
+        <AdditionalFeatures />
+        <Total />
       </div>
     </div>
   );
```

Comment:
You'll need mapStateToProps and connect here, and then plug your props into your components. Or, you can do this at the child component level. If you're doing this on the child component level, then you should put your remove feature and buy item functions in the appropriate components.